controllers/seasons.controllers.ts

```diff
@@ -1,7 +1,116 @@
-// TODO: require the model
-// Season = require("../models/Season")
+import { NextFunction, Response, Request } from "express";
+import Season from "../models/Season";
 
-// TODO: add controller logic 
+function create(req: Request, res: Response, next: NextFunction) {
+  const season = new Season({
+    name: req.body.name,
+    startDate: req.body.startDate,
+    endDate: req.body.endDate, //Think this should be calculated when status is changed to close
```

Comment:
yes, I think you are right, 
Let's leave the endDate as optional so we can create harvest seasons without the endDate, and set the endDate default to now inside the close season endpoint.

Reply:
https://github.com/LangaraVoyagers/project-2-api/pull/8/commits/0764e617ad904de9c34bc9c5751d2fbdb6874b9c